weatherApp/Features/WeatherDetails/UI/DetailWeatherViewController.swift

```diff
@@ -44,19 +51,33 @@ class DetailWeatherViewController: UIViewController {
         view.setGradientBackground(startColor: .grayBlueTint, endColor: .darkNavyBlue)
     }
     
-    private func bindViewModel() {
-        guard let detailWeatherPresenter = detailWeatherPresenter else { return }
-        
+    //MARK: - Data
+    
+    @objc private func bindViewModel() {
         detailWeatherPresenter.fetchFiveDaysList() { (currentWeatherList) in
-            guard let _ = currentWeatherList else {
-                return
-            }
+            guard let _ = currentWeatherList else { return }
+            
+            self.refreshCollectionViewData()
+        }
+        
+        detailWeatherPresenter.fetchCurrentWeather { (currentWeather) in
+            guard let _ = currentWeather else { return }
+            
             DispatchQueue.main.async {
-                self.daysCollectionView.reloadData()
+                self.setWeatherInformation()
             }
         }
     }
     
+    private func startTimer() {
+        timerObservable = Observable<Int>
+            .timer(.seconds(0), period: .seconds(300), scheduler: MainScheduler.instance)
```

Comment:
I think that you have .minutes also. Maybe it will be better to use minutes. 
Also, you can move minutes value into some property which can be configured later.

Reply:
The biggest unit for RxTimeInterval is seconds unfortunately.